movies/src/tests/tests.hydra

```diff
@@ -0,0 +1,34 @@
+PREFIX schema: <http://schema.org/>
+PREFIX hydra: <http://www.w3.org/ns/hydra/core#>
+PREFIX mov: <https://hydra-movies.herokuapp.com/doc#>
+
+With Class schema:EntryPoint {
+    Expect Property hydra:collection {
+        Expect Type hydra:Collection
+        Expect Type mov:Movies
+    }
+
+    Expect Link hydra:collection {
```

Comment:
This currently fails because hydra:collection is not a link.

Should the hydra DSL flavour be extended with something like `Expect Collection mov:Movies`?

Reply:
Hydra vocabulary clearly states that `hydra:collection` is a `hydra:Link`. Whatever tool performs the assertion regarding hydra vocabulary must understand it's semantics.

Reply:
> Hydra vocabulary clearly states that hydra:collection is a hydra:Link

Quite right. The client is wrong here 😢